web/static/js/marked/lib/marked.js

```diff
@@ -0,0 +1,980 @@
+/**
```

Comment:
I modified it, since it's no longer markdown I had to go in and change how a lot of the parsing was done (ripped out most of the features and changed bold and italics to be only single asterisks and single underscores respectively).